### PR TITLE
[PUB-801] Display custom footer message when IG reminder

### PR DESCRIPTION
**Pull request by anafilipadealmeida**
### Purpose
Add a custom message on the footer of each Instagram Reminder Post

### CODE REVIEW

File: packages/shared-components/PostFooter/index.jsx [hunk 1]
```diff
@@ -65,6 +56,16 @@ const renderText = ({ postDetails }, hasError, isSent) =>
     </Text>
   </span>);
 
+const renderIcon = (hasError, isSent, isCustomScheduled, isInstagramReminder) => {
+  if (!hasError && !isCustomScheduled && !isInstagramReminder) return;
+
+  return (<div style={postActionDetailsIconStyle}>
+    {hasError ? <WarningIcon color={'torchRed'} /> : null}
+    {isInstagramReminder && !hasError ? <CircleInstReminderIcon color={'instagram'} /> : null}
```
Comment (karelm) [inline packages/shared-components/PostFooter/index.jsx]: hey @anafilipadealmeida is this meant to be color={'instagram'} here? just checking 😄

File: packages/shared-components/PostFooter/index.jsx [hunk 2]
```diff
@@ -83,10 +84,10 @@ const PostFooter = ({
 }) => {
   const hasError = postDetails.error && postDetails.error.length > 0;
   const isCustomScheduled = postDetails.isCustomScheduled;
+  const isInstagramReminder = postDetails.isInstagramReminder;
   return (<div style={isSent? sentPostDetailsStyle : getPostDetailsStyle(dragging)}>
     <div style={postActionDetailsStyle}>
-      {hasError && renderErrorIcon()}
-      {isCustomScheduled && !hasError && renderCustomScheduledIcon(isSent)}
+      {renderIcon(hasError, isSent, isCustomScheduled, isInstagramReminder)}
```
Comment (karelm) [inline packages/shared-components/PostFooter/index.jsx]: thats so much tidier! 👍

File: packages/shared-components/PostFooter/index.jsx (hunk 1 above)
Comment (anafilipadealmeida) [reply packages/shared-components/PostFooter/index.jsx]: Yes 🤣 It's equivalent to #e53c5f, you can see the full list on `buffer-components/style/color.js` !

File: packages/utils/postParser.js [hunk 3]
```diff
@@ -28,9 +28,14 @@ const getPostActionString = ({ post }) => {
     },
   );
 
-  if (post.scheduled_at) {
+  if (post.scheduled_at && post.can_send_direct !== false) {
```
Comment (karelm) [inline packages/utils/postParser.js]: does this need to use !== false as its kind of hard to read a double negative... can this be simplified with a !!post.can_send_direct ?

Comment (karelm) [reply packages/utils/postParser.js]: just an addition here `undefined !== false` and `null !== false`... just wanted to check that with you that you are looking for anything other than explicitly false here

Comment (anafilipadealmeida) [reply packages/utils/postParser.js]: I wanted this to run with any other values, just not with false. Sometimes this property is undefined, and I still wanted it to run in that situation.

**Pull request merged**